admin_manual/configuration/server/security/password-policy.rst

```diff
@@ -8,23 +8,34 @@ Password Policy
 ---------------
 
 .. figure:: ../../../images/configuration/server/security/password-policy-app.png
-   :alt: The (enterprise-only) password policy application
+   :alt: The Password Policy application
 
-ownCloud Enterprise users have the option of enabling `the Password Policy app`_. 
-This application enables administrators to define password requirements, including: *minimum characters*, *numbers*, *capital letters*, and more, for all kinds of password endpoints, such as for user passwords and public link sharing passwords. 
-Moreover the admin can enforce expiration dates for public link shares.
+From the 2.0.0 release of `the Password Policy app`_, all ownCloud users (both enterprise and community) have the option of installing and enabling the application.
```

Comment:
``all ownCloud users (both enterprise and community) have the option`` -->
``both enterprise and community editions have the option``

It is dangerous to write ``users`` even true. This makes admins nervous ! 😄